ci/scripts/gen-flamegraph.sh

```diff
@@ -209,7 +209,7 @@ gen_events() {
 }
 
 show_kafka_topics() {
-  "${KAFKA_DIR}"/bin/kafka-run-class.sh kafka.tools.GetOffsetShell --topic nexmark --bootstrap-server localhost:9092
+  "${KAFKA_DIR}"/bin/kafka-get-offsets.sh --topic nexmark --bootstrap-server localhost:9092
```

Comment:
have been using `kafka-run-class.sh kafka.tools.GetOffsetShell` for a long time and was not aware of `get-offsets.sh`

Reply:
Yeah looks like it got deprecated or something not sure. It failed when I tried it on ;latest kafka version. This seems to work